server/migrations/008.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE if not exists project_access (
+    id serial,
+    project_id text not null references project(proj_id) ON DELETE CASCADE,
```

Comment:
Does this imply that there will be multiple entries for each project?

Reply:
no - there is actually a unique constraint for it (`ADD CONSTRAINT "unique_project_access" UNIQUE ("project_id")`)